pyproject.toml

```diff
@@ -12,6 +12,7 @@ anyio = ">=4.0.0"
 async-asgi-testclient = ">=1.4.11,<2.0.0"
 bokeh = ">=3.1.0,<3.2.0"
 cai-causal-graph = ">=0.3.6"
+certifi = ">=2024.7.4"
```

Comment:
That's just the top-level pyproject which will be regenerated by anthology. The minimum version should be set in the packages themselves.

Reply:
Oops, good catch fixed it